frontend/src/App.tsx

```diff
@@ -36,6 +37,11 @@ const App = () => {
               path="/p/:version/:id"
               component={ProtectedPage}
             ></Route>
+            <Route
+              exact
+              path="/unsubscribe/:version"
```

Comment:
out of curiosity, why did we choose to use `/unsubscribe/:version?c=&r=&h` instead of  `/unsubscribe?v=&c=&r=&h`

Reply:
Mainly chose this to keep it consistent with `ProtectedPage`.